kernel/src/fs/cgroupfs/systree_node.rs

```diff
@@ -542,3 +657,58 @@ where
 
     op(process, &mut cgroup_guard)
 }
+
+/// Reads the actions for sub-control from the given reader.
+///
+/// Returns the sets of controllers to be activated and deactivated,
+/// along with the number of bytes read. The activeate and deactivate sets
```

Comment:
`The activeate and deactivate sets` -> `The two sets`

"activeate" is a typo, anyway.